apps/api-gateway/src/app.module.ts

```diff
@@ -2,6 +2,7 @@ import { Logger, MiddlewareConsumer, Module, NestModule } from '@nestjs/common';
 import { ConfigModule, ConfigService } from '@nestjs/config';
 import { MongooseModule, MongooseModuleOptions } from '@nestjs/mongoose';
 import { TypeOrmModule, TypeOrmModuleOptions } from '@nestjs/typeorm';
+import { ChatModule } from 'apps/chat/src/chat/chat.module';
```

Comment:
Microservice chat과 연결하기 위해선 우선 api-gateway에 있는 chat-module을 사용해야할 것 같습니다!

```suggestion
import { ChatModule } from './chat/chat.module';
```

Reply:
잘못된 import 설정을 했네요. 수정하겠습니다!